tutorials/stats/2_pipeline_integration.py

```diff
@@ -0,0 +1,111 @@
+# %% [markdown]
+"""
+# 2. Pipeline Integration
+
+In the DFF ecosystem, extractor functions act as regular extra handlers (
+[see the pipeline module documentation](
+https://deeppavlov.github.io/dialog_flow_framework/tutorials/tutorials.pipeline.7_extra_handlers_basic.html
```

Comment:
Use new syntax of documentation linking (if merged after #136).

Reply:
Done